qa/logging-config/src/test/java/org/elasticsearch/common/logging/ESJsonLayoutTests.java

```diff
@@ -25,45 +27,21 @@ public void testLayout() {
         ESJsonLayout server = ESJsonLayout.newBuilder().setType("server").build();
         String conversionPattern = server.getPatternLayout().getConversionPattern();
 
-        assertThat(
-            conversionPattern,
-            Matchers.equalTo(
-                "{"
-                    + "\"type\": \"server\", "
-                    + "\"timestamp\": \"%d{yyyy-MM-dd'T'HH:mm:ss,SSSZZ}\", "
-                    + "\"level\": \"%p\", "
-                    + "\"component\": \"%c{1.}\", "
-                    + "\"cluster.name\": \"${sys:es.logs.cluster_name}\", "
-                    + "\"node.name\": \"%node_name\", "
-                    + "\"message\": \"%notEmpty{%enc{%marker}{JSON} }%enc{%.-10000m}{JSON}\""
-                    + "%notEmpty{, %node_and_cluster_id }"
-                    + "%notEmpty{, %CustomMapFields }"
-                    + "%exceptionAsJson }"
-                    + System.lineSeparator()
-            )
-        );
+        assertThat(conversionPattern, Matchers.equalTo(String.format(Locale.ROOT, """
+            {"type": "server", "timestamp": "%%d{yyyy-MM-dd'T'HH:mm:ss,SSSZZ}", "level": "%%p", "component": "%%c{1.}", \
```

Comment:
Can we split this over multiple lines, to make it more readable? Same thing below as well.